lib/worker.py

```diff
@@ -148,12 +148,13 @@ class WorkerTaskResult(BaseWorkerMessage):
     the processed task. The show_reproduce_content configuration form suite.ini
     """
     def __init__(self, worker_id, worker_name, task_id,
-                 short_status, result_checksum, show_reproduce_content):
+                 short_status, result_checksum, is_long, show_reproduce_content):
         super(WorkerTaskResult, self).__init__(worker_id, worker_name)
         self.short_status = short_status
         self.result_checksum = result_checksum
         self.task_id = task_id
         self.show_reproduce_content = show_reproduce_content
+        self.is_long = is_long
```

Comment:
Nit: Please, follow the same order in parameters and those assignments. It eases reading.

Reply:
Right, corrected.